### PR TITLE
Update server port to 3002

### PR DESCRIPTION
**Pull request by flomillot**
Dependents with gridsuite/deployment#401

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
         "yup": "^1.2.0"
     },
     "scripts": {
-        "start": "react-scripts start",
+        "start": "PORT=3003 react-scripts start",
```
Comment (Tristan-WorkGH) [inline package.json:53]:
```suggestion
        "start": "PORT=3002 react-scripts start",
```

File: public/idpSettings.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
     "authority": "http://172.17.0.1:9090/",
-    "client_id": "my-client-2",
-    "redirect_uri": "http://localhost:3000/sign-in-callback",
-    "post_logout_redirect_uri": "http://localhost:3000/logout-callback",
-    "silent_redirect_uri": "http://localhost:3000/silent-renew-callback",
+    "client_id": "gridadmin-local",
+    "redirect_uri": "http://localhost:3003/sign-in-callback",
+    "post_logout_redirect_uri": "http://localhost:3003/logout-callback",
+    "silent_redirect_uri": "http://localhost:3003/silent-renew-callback",
```
Comment (Tristan-WorkGH) [inline public/idpSettings.json:6]:
```suggestion
    "redirect_uri": "http://localhost:3002/sign-in-callback",
    "post_logout_redirect_uri": "http://localhost:3002/logout-callback",
    "silent_redirect_uri": "http://localhost:3002/silent-renew-callback",
```

**Review by Tristan-WorkGH — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (flomillot) [reply package.json]: Done

File: public/idpSettings.json (hunk 2 above)
Comment (flomillot) [reply public/idpSettings.json]: Done

**Review by Tristan-WorkGH — APPROVED** [review]

**Review by EstherDarkish — APPROVED** [review]

**Pull request merged**